File: 122. Best Time to Buy and Sell Stock II/Python/Solution.py
class Solution:
    def maxProfit(self, prices):
        """
        :type prices: List[int]
        :rtype: int
        """
        profit = 0
        for i in range(1, len(prices)):
            gain = prices[i] - prices[i - 1]
            if gain > 0:
                profit += gain
        return profit

File: 122. Best Time to Buy and Sell Stock II/Java/Solution.java
public class Solution {
    public int maxProfit(int[] prices) {
        int profit = 0;
        for (int i = 1; i < prices.length; i++) {
            int gain = prices[i] - prices[i - 1];
            if (gain > 0) {
                profit += gain;
            }
        }
        return profit;
    }
}